stampy-check.yaml

```diff
@@ -0,0 +1,227 @@
+resources:
+- name: src
+  type: pull-request
+  source:
+    access_token: {{github-access-token}}
+    branch: master
+    private_key: {{github-private-key}}
+    uri: git@github.com:SUSE/stampy.git
+
+- name: status
+  type: status
+  source:
+    access_token: {{github-access-token}}
+    repo: SUSE/stampy
+
+- name: src-ci
+  type: git
+  source:
+    branch: master
+    private_key: {{github-private-key}}
+    uri: git@github.com:SUSE/stampy-ci.git
+
+- name: stampy-binary-linux
+  type: s3
+  source:
+    access_key_id: {{s3-access-key}}
+    endpoint: {{s3-endpoint}}
+    secret_access_key: {{s3-secret-key}}
+    bucket: stampy
+    regexp: stampy-(.*\.g.*\).linux.amd64\.tgz
```

Comment:
Oops, the backslash is in the wrong place (should be before the dot, not before the closing parenthesis)